common/constants.sh

```diff
@@ -17,6 +17,7 @@ export STARTUP=0
 export SYNC_GATEWAY=0
 export WAIT=0
 export NO_CLUSTER=0
+export CLUSTER_ONLY=0
```

Comment:
Probably could do with documenting these as not overly clear with, no, do and only

Reply:
Yep.. added